datadog/resource_datadog_dashboard.go

```diff
@@ -2976,6 +3046,43 @@ func getTimeseriesRequestSchema() map[string]*schema.Schema {
 		"log_query":     getApmOrLogQuerySchema(),
 		"process_query": getProcessQuerySchema(),
 		// Settings specific to Timeseries requests
+		"style": {
+			Type:     schema.TypeList,
+			Optional: true,
+			MaxItems: 1,
+			Elem: &schema.Resource{
+				Schema: map[string]*schema.Schema{
+					"palette": {
+						Type:     schema.TypeString,
+						Optional: true,
+					},
+					"line_type": {
+						Type:     schema.TypeString,
+						Optional: true,
+					},
+					"line_width": {
+						Type:     schema.TypeString,
+						Optional: true,
+					},
+				},
+			},
+		},
+		"metadata": {
+			Type:     schema.TypeList,
+			Optional: true,
+			Elem: &schema.Resource{
+				Schema: map[string]*schema.Schema{
+					"expression": {
+						Type:     schema.TypeString,
+						Optional: true,
```

Comment:
Should this be required (the docs changes say this field is required)

Reply:
You're absolutely correct. It is a required. Thanks for catching this.